projects.php

```diff
@@ -0,0 +1,43 @@
+<?php
+require_once './database.php';
+
```

Comment:
you could refactor this into functions eventually :)

Reply:
Will do!